src/mysql.erl

```diff
@@ -142,6 +143,16 @@
 %%   </dd>
 %%   <dt>`{connect_timeout, Timeout}'</dt>
 %%   <dd>The maximum time to spend for start_link/1.</dd>
+%%   <dt>`{allowed_local_paths, [binary()]}'</dt>
+%%   <dd>This option allows you to specify a list of directories or individual
+%%       files on the client machine which the server may request, for example
+%%       when executing a `LOAD DATA LOCAL INFILE' query. Only absolute paths
+%%       without relative components such as `..' and `.' are allowed.
+%%       A path ending with a `/' denotes a directory from which files
```

Comment:
If we handle dirs with or without slash, we don't need this sentence. No need to distinguish between dirs and regular files.

Reply:
... and removed the sentence.